src/bcftools/bcftools_concat/config.vsh.yaml

```diff
@@ -27,124 +27,201 @@ argument_groups:
   - name: Inputs
     arguments:
       - name: --input
-        alternatives: -i
+        alternatives: [-i]
```

Comment:
I noticed that sometimes alternatives is formated as `alternatives: [-a]` and sometimes as `alternatives: -a`